src/ott/geometry/regularizers.py

```diff
@@ -305,55 +335,41 @@ def tree_flatten(self):  # noqa: D102
 
 @jtu.register_pytree_node_class
 class L1(ProximalOperator):
-  r"""L1 norm regularizer :math:`\lambda \|x\|_1`.
-
-  Args:
-    lam: Strength of the regularizer.
-  """
-
-  def __init__(self, lam: float = 1.0):
-    super().__init__()
-    self.lam = lam
+  r"""L1 norm regularizer :math:`\|x\|_1`."""
 
   def __call__(self, x: jnp.ndarray) -> float:  # noqa: D102
-    return self.lam * jnp.linalg.norm(x, ord=1)
+    return jnp.linalg.norm(x, ord=1)
 
   def prox(self, v: jnp.ndarray, tau: float = 1.0) -> jnp.ndarray:  # noqa: D102
-    return jnp.sign(v) * jax.nn.relu(jnp.abs(v) - self.lam * tau)
-
-  def tree_flatten(self):  # noqa: D102
-    return (self.lam,), {}
+    return jnp.sign(v) * jax.nn.relu(jnp.abs(v) - tau)
 
 
 @jtu.register_pytree_node_class
 class L2(ProximalOperator):
-  r"""L2 norm regularizer :math:`\frac{\lambda}{2} \left<x, A^TAx\right>`.
+  r"""L2 norm regularizer :math:`\frac{1}{2} \left<x, A^TAx\right>`.
```

Comment:
same, and we might want to call this `:math:\ell_2^2` i.e. `L22`, or `SqL2`

Reply:
I'd keep it as `L2`, as it seems also consistent with literature (see, e.g., [here](https://icml.cc/Conferences/2004/proceedings/papers/354.pdf)).

Reply:
indeed, this is also used here: https://pyproximal.readthedocs.io/en/stable/api/generated/pyproximal.L2.html#pyproximal.L2

but because we make the distinction between `Euclidean` and `SqEuclidean` I feel the `Sq` would be more rigorous and help maintain consistency.